src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs

```diff
@@ -121,15 +122,18 @@ public virtual void SetupChannel(IEnumerable<string> sources)
                     };
                 }
 
+                // Launch the test host.
+                var testHostProcessId = this.testHostManager.LaunchTestHost(testHostStartInfo);
+
                 // Warn the user that execution will wait for debugger attach.
                 var hostDebugEnabled = Environment.GetEnvironmentVariable("VSTEST_HOST_DEBUG");
                 if (!string.IsNullOrEmpty(hostDebugEnabled) && hostDebugEnabled.Equals("1", StringComparison.Ordinal))
                 {
                     ConsoleOutput.Instance.WriteLine(CrossPlatEngineResources.HostDebuggerWarning, OutputLevel.Warning);
+                    ConsoleOutput.Instance.WriteLine(
+                    string.Format("Process Id: {0}, Name: {1}", processId, Process.GetProcessById(testHostProcessId).ProcessName),
```

Comment:
Please don't use Process directly. Is there a core utilities abstraction for Process that can be used? Or something in PlatformAbstractions?